packages/seacas/libraries/ioss/src/cgns/Iocgns_DatabaseIO.C

```diff
@@ -1888,14 +1885,14 @@ namespace Iocgns {
           // End of lambda...
           // ========================================================================
 
-          blk_coord_lambda("CoordinateX", 0, get_file_pointer(), myProcessor);
+          blk_coord_lambda("CoordinateX", 0, get_file_pointer());
```

Comment:
Could potentially access `get_file_pointer()` internal to lambda now that `this` is being passed in...  (something to look at for later)